ThirteenLtda/thirteen.squidbot-package_set
# Use rock-core base-types instead of thirteen's fork

This PR reverts the base-types library to use rock-core package instead of https://github.com/ThirteenLtda/base-types. This fixes a build error due to incompatibility between `base-orogen-types` and `base-types` since the first was using rock-core repo while the second was using thirteen's fork.

@sbolivarm, I see that this fork has been created here https://github.com/ThirteenLtda/base-types/pull/1? Is there any reason for us to have this fork, if so, I think we would also need to fork https://github.com/rock-core/base-orogen-types to avoid new issues like this one. The test procedure below has been executed and no errors were found though.  

How to test
--------------

* Execute the following commands
```bash
acd
aup
amake
```
* Check if the simulation is still working fine

```bash
# Terminal 1
syskit ide -rgazebo
```
```bash
# From syskit ide
inspect_sector! sector_id: 0
```
